source/deployment/mobile-app-deployment.rst

```diff
@@ -15,67 +15,57 @@ Leaders, Administrators, and/or Champions
+If at any point you need additional help, please reach out using the methods below.
 
 **Community**
-    Our entire team is active within the `public instance of Mattermost <https://community.mattermost.com>`_, plus you’ll have the support of one of the best open source communities around.
+    Our entire team is active within the `public instance of Mattermost <https://community.mattermost.com>`_, plus you will have the support of one of the best open source communities around.
 **Documentation**
     We link to a lot of `mobile-specific documentation <https://docs.mattermost.com/mobile/mobile-overview.html>`_ within this guide, but we encourage you to check `all of our documentation <https://docs.mattermost.com/>`_.
 **Forums**
     For more troubleshooting help, `open a new topic in our forums <https://forum.mattermost.org/c/trouble-shoot>`_ with steps to reproduce your issue.
 **GitHub**
     Visit us on `GitHub <https://github.com/mattermost/>`_ to create issues in any of our repositories.
 **Enterprise Support**
-    If you’re an Enterprise Edition subscriber, you may open a support ticket in the `Enterprise Edition Support portal <https://mattermost.com/support/>`_.
+    If you are an Enterprise Edition subscriber, you may open a support ticket in the `Enterprise Edition Support portal <https://mattermost.com/support/>`_.
 
 ----
 
 ################################
-Before You Begin Your Deployment
+Before you begin your deployment
 ################################
 
 A successful mobile deployment is an important part of your experience with Mattermost.
 
-As with any new project, a variety of risks can limit the success of adoption. This translates to wasted time, unnecessary cost, and unhappy teams. Any change, even beneficial, will have an impact on your users. 
-
-For this reason, our highest priority is making sure you have what you need to be successful. That starts with helping you identify the outcomes you’re looking to achieve.
-
-Another way of looking at this is to consider the goals for your mobile deployment.
-
-For example, do you want the simplest, fastest deployment possible?
-
-Using the publicly available Mattermost mobile apps on Google Play and Apple App Store can reduce a deployment to minutes.
-
-Better yet, this model can be rolled out to thousands of users in many different countries and languages.
+Our highest priority is making sure you have what you need to be successful, but keep in mind that any change, even beneficial, will have an impact on your users. To ensure your success you need to identify the goals for your mobile deployment.
 
-Of course, large enterprise deployments will take more time, and the overall goals for your deployment will depend on many factors.
+For example, using the publicly available Mattermost mobile apps on Google Play and Apple App Store can reduce a deployment to minutes. Better yet, this model can be rolled out to thousands of users in many different countries and languages.
```

Comment:
We can write more concisely by being direct, without losing the intended tone to the reader

```suggestion
For example, you can deploy to thousands of users across the world in minutes with the publicly available Mattermost mobile apps on Google Play and Apple App Store.
```